docs/en/latest/plugins/openwhisk.md

```diff
@@ -23,34 +28,36 @@ title: openwhisk
-| keepalive_pool | integer | optional | 5 | [1,...] | Connection pool limit |
+| Name              | Type    | Required | Default | Valid values | Description                                                                                                |
+|-------------------|---------|----------|---------|--------------|------------------------------------------------------------------------------------------------------------|
+| api_host          | string  | True     |         |              | OpenWhisk API host address. For example, `https://localhost:3233`.                                         |
+| ssl_verify        | boolean | False    | true    |              | When set to `true` verifies the SSL certificate.                                                           |
+| service_token     | string  | True     |         |              | OpenWhisk service token. The format is `xxx:xxx` and it is passed through basic auth when calling the API. |
+| namespace         | string  | True     |         |              | OpenWhisk  namespace. For example `guest`.                                                                 |
+| action            | string  | True     |         |              | OpenWhisk action. For example `hello`.                                                                     |
+| result            | boolean | False    | true    |              | When set to `true` gets the action metadata (executes the function and gets response).                     |
+| timeout           | integer | False    | 60000ms | [1, 60000]ms | OpenWhisk action and HTTP call timeout in ms.                                                              |
+| keepalive         | boolean | False    | true    |              | When set to `true` keeps the connection alive for reuse.                                                   |
+| keepalive_timeout | integer | False    | 60000ms | [1000,...]   | Time is ms for connection to remain idle without closing.                                                  |
+| keepalive_pool    | integer | False    | 5       | [1,...]      | Maximum number of requests that can be sent on this connection before closing it.                          |
 
 :::note
 
-- The `timeout` property controls both the time taken by the OpenWhisk Action to execute and the timeout of the HTTP client in APISIX. OpenWhisk Action calls may consume time on pulling the runtime image and starting the container, so if you set the value too small, you may cause a large number of requests to fail. OpenWhisk supports timeouts ranging from 1ms to 60000ms, and we recommended to set at least 1000ms or more.
+The `timeout` attribute sets the time taken by the OpenWhisk action to execute and also the timeout for the HTTP client in APISIX. OpenWhisk action calls may take time to pull the runtime image and start the container. So, if the value is set too small, it may cause a large number of requests to fail.
```

Comment:
```suggestion
The `timeout` attribute sets the time taken by the OpenWhisk action to execute and the timeout for the HTTP client in APISIX. OpenWhisk action calls may take time to pull the runtime image and start the container. So, if the value is set too small, it may cause a large number of requests to fail.
```

Reply:
Updated.